parl/utils/exceptions.py

```diff
@@ -28,6 +28,9 @@ class SerializeError(UtilsError):
     """
 
     def __init__(self, error_info):
+        error_info = (
+            'Serialize error, you may have provided a pyarrow ' +
```

Comment:
you may have provided an object that cannot be serialized by pyarrow.

Reply:
done